sky/global_user_state.py

```diff
@@ -134,6 +140,7 @@ def colored_str(self):
     ClusterStatus.INIT: colorama.Fore.BLUE,
     ClusterStatus.UP: colorama.Fore.GREEN,
     ClusterStatus.STOPPED: colorama.Fore.YELLOW,
+    ClusterStatus.TERMINATED: colorama.Fore.RED,
```

Comment:
It would be a bit alerting to use `RED`. It could be better to use some less noticeable color, e.g. `GREY` instead?